cmd/init.go

```diff
@@ -45,7 +52,8 @@ func (cli *CLI) init(opts Options) int {
 
 				_, err := plugin.FindPluginPath(installCfg)
 				if os.IsNotExist(err) {
-					fmt.Fprintf(cli.outStream, "Installing \"%s\" plugin...\n", pluginCfg.Name)
+					installed = true
+					builder.WriteString(fmt.Sprintf("Installing \"%s\" plugin...\n", pluginCfg.Name))
```

Comment:
This message must be output in real time. It doesn't make sense to see "Installing..." printed after the installation is complete.

Reply:
Sounds good. I updated it to immediately print the builder string and then send the rest of the output directly to stdout when a plugin is installed (or tried to install)